lib/psdk/cli.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'thor'
+require_relative 'cli/version'
+require_relative 'cli/plugin'
+
+module Psdk
+  module Cli
+    # Main PSDK CLI class
+    #
+    # Must be used for the general psdk-cli command
+    class Main < Thor
+      package_name 'psdk-cli'
+
+      desc('version', 'show the psdk-cli version')
+      def version
+        puts "psdk-cli v#{VERSION}"
+      end
+
+      desc 'plugin', 'manage PSDK plugins'
```

Comment:
Is there a reason why there is no parenthesis on this `desc`?

Reply:
Faudrait que je retire les parenthèses sur le précédent.

Quand c'est du DSL faut pas mettre de parenthèses sauf si y'a des blocs et des arguments en même temps. C'est aussi comme ça que Thor documente les choses donc vaut mieux faire pareil si rubocop ne dit rien.

https://github.com/rails/thor/wiki/Subcommands